app/components/appMainModule/localePageModule/electionTypePage/electionTypePageRoutes.js

```diff
@@ -5,7 +5,13 @@ function electionTypePageRoutes($stateProvider) {
   var electionTypePage = {
     name: 'appMain.localePage.electionTypePage',
     url: '/:electionYear/:electionType/:electionTypeId/:electionTitle',
-    // url: '/:electionYear/:electionType',
+    abstract: true,
+    template: '<div ui-view></div>'
```

Comment:
Yeah, it acts as a wrapper/container because of the `ui-view`. As far as I understand, `abstract` only makes the state not routable. Everything else still applies e.g. controller, url, template, resolve, even breadcrumbs (although [abstract states are ignored by default](http://angular-ui.github.io/ui-router/site/#/api/ui.router.state.$stateProvider)).

Notice how `/city/2/oakland` must have a `ui-view` so that child states have a view to render into. But when you're on `/city/2/oakland`, there is no child state, hence no scope, hence the broken breadcrumb again. I forgot that making `appMain.localePage` abstract was part of the breadcrumb fix.